lib/widgets/app/initializing_indicator.dart

```diff
@@ -18,9 +18,9 @@ class InitializingIndicator extends StatelessWidget {
   Widget build(BuildContext context) {
     var message = '';
     if (!firebaseDone) {
-      message = 'Waiting for Firebase...';
+      message = 'Enticing a ghost into the machine...';
     } else if (!reduxDone) {
-      message = 'Waiting for Redux...';
+      message = 'Plumbing the pipes...';
```

Comment:
works as an abstract visualization for the user to understand whats going on and for us devs as a literal point to look at,
cause this redux structure has got some pipage

Reply:
Awesome, that' s what I was hoping

Reply:
Thanks!